docs/hardware/monitor.md

```diff
@@ -98,21 +98,6 @@ mounting. Before installation, take note of the following pre-installation preca
 - Mount the monitor at normal human breathing height (about 1.5-2m above the
   ground)
 
```

Comment:
What is the motivation for removing these scoping requirements? @NsubugaAnold 

cc: @123MwanjeMike

Reply:
That is the current state of the manual as @NsubugaAnold  can confirm @Baalmart

Reply:
Alright, thanks @123MwanjeMike